Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -1707,6 +1725,25 @@ public override void Initialize()
                 }
             
             }
+
+            // Check to see if water scoop elements have been configured
+            if (WaterScoopFillElevationM == 0)
+            {
+               WaterScoopFillElevationM = 2.7432f; // Set to default of 9 ft
+            }
+            
+            if (WaterScoopDepthM == 0)
+            {
+               WaterScoopDepthM = 0.0889f; // Set to default of 3.5 ins
+            }
+            
+            if (WaterScoopWidthM == 0)
+            {
+               WaterScoopWidthM = 0.3048f; // Set to default of 1 ft
+            }
+
+            // Calculate minimum speed to pickup water
+               WaterScoopMinSpeedMpS = Me.FromFt((float)Math.Sqrt(2 * 32.2 * Me.ToFt(WaterScoopFillElevationM)));
```

Comment:
This maths expression is inscrutable; please either use constants for the numbers (so I and others know what they mean/represent) or at least wrap the calculation inside a function with a clear name.

Reply:
I don't quite understand the issue with this statement, can you expand on the issue for me.

Reply:
The numeric constants `2` and `32.2` mean nothing to me or anyone else looking at this code. If you put them into `const` fields, you can give them readable names *and* units - both of which are highly useful when reading the code. It is probably also worth commenting what the maths is here - why are we doing a square-root on a * b * c? What real-world thing is this?

I am separately sad that this converts metres into feet, does the maths, and then puts it back into metres - that probably helps nobody in this particular case, but definitely will make the calculation less accurate (every conversion makes things less accurate because computers can't represent floating point numbers exactly).